src/components/navBar/navBar.module.scss

```diff
@@ -8,28 +8,71 @@ $thickness: 3px;
   align-items: center;
   overflow: hidden;
   flex-wrap: wrap;
+  justify-content: space-between;
 
-  a {
-    display: block;
-    color: #ffffff;
-    font-weight: bolder;
-    text-align: center;
-    padding: 35px 2*($offset+$thickness)-1;
-    text-decoration: none;
+  div {
+    display: flex;
 
-    &:hover {
+    a {
+      display: block;
+      color: #ffffff;
+      font-weight: bolder;
+      text-align: center;
+      padding: 35px 2 * ($offset + $thickness)-1;
+      text-decoration: none;
+
+      &:hover {
+        color: #87d870;
+      }
+    }
+
+    .active {
       color: #87d870;
+      text-decoration: underline;
+      text-decoration-thickness: $thickness;
+      text-underline-offset: $offset;
     }
```

Comment:
can we put all the colors in css variable and then use it here

Reply:
Wouldn't it be better to create a separate issue for this?

Reply:
I guess you can do it here, there is no need for another PR for this

Reply:
I am not sure why it is required. Can we discuss more on it on discord?

Reply:
Read [this](https://developer.chrome.com/blog/css-variables-why-should-you-care/)